src/Actus/Domain.purs

```diff
@@ -97,32 +98,55 @@ instance ActusOps Value' where
 instance ActusFrac Value' where
   _ceiling _ = 0 -- FIXME
 
-marloweFixedPoint :: BigInt
-marloweFixedPoint = fromInt 1000000
+derive instance Generic Value' _
+derive instance Generic Observation' _
+
+instance Show Value' where
+  show (Constant' i) = "Constant " <> show i
+  show (NegValue' v) = "-" <> show v
```

Comment:
if `v` is `AddValue x y`, this would produce p `-AddValue x y` which I don't think it would parse